app/controllers/comments_controller.rb

```diff
@@ -1,4 +1,42 @@
 class CommentsController < AuthenticatedController
+  include ActionView::RecordIdentifier
   include ProjectScoped
-end
 
+  before_action :find_or_initialize_comment
+  before_action :check_comment_author, only: [:update, :destroy]
```

Comment:
Instead of checking this and returning a blank screen (what `head` does in a browser), we've pre-empted the problem by assigning the model's user after the fact:

https://github.com/secroots/dradis-pro/blob/develop/app/controllers/notes_controller.rb#L18

Also, :user_id isn't included in `comment_params` which I believe means negates this problem?

I'd keep `:user_id` excluded, but also explicitly add a `comment.user = current_user` in #create

Reply:
@etdsoft this check is for the `:destroy` and `:update actions` only.
I wanted to prevent `current_user` from editing or deleting comments  if she/he is not the author of that comment.

Reply:
then I'd rename to `:ensure_comment_author` and handle it properly instead of bailing with a `head`.